app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_like_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [optional] I think you can make this method to be private because we only need to call this method from inside this class where it's defined (  **You can implement the same for `comment and post`models as well.**).
- [optional] Kindly use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the #update_like_counter method every time a new like has been saved to database. ℹ️  **You can implement the same for `post `model as well.**